include/enum_quda.h

```diff
@@ -18,18 +18,18 @@ typedef enum QudaMemoryType_s {
   QUDA_MEMORY_INVALID = QUDA_INVALID_ENUM
 } QudaMemoryType;
 
-//
-// Types used in QudaGaugeParam
-//
+/**
```

Comment:
Also here I changed all the comments to C89 style ones to be able to compile openQxD in C89. People don't want to change that standard ...